allvars.c

```diff
@@ -30,6 +30,10 @@ int RestartFlag;		/*!< taken from command line used to start code. 0 is normal s
 				   marks a restart from a snapshot file. */
 int RestartSnapNum;
 
+/* paired/fixed simulation flags*/
+int PairedSim;
+int FixedAmpSim;
+
```

Comment:
You don't need to add these parameters to allvars.[ch] if you have already added them to genic/allvars.[ch]: they are only used when generating initial conditions.